File: prev codes/TCPClient.java
import java.net.*;
import java.io.*;

public class TCPClient {
    public static void main(String[] args) throws IOException {
        Socket socket = new Socket("127.0.0.1", 5001);
        BufferedReader in = new BufferedReader(new InputStreamReader(socket.getInputStream()));
        PrintWriter out = new PrintWriter(socket.getOutputStream(), true);
        BufferedReader userIn = new BufferedReader(new InputStreamReader(System.in));

        System.out.println("Connected to the server!");
        String clientMsg, serverMsg;
        while (true) {
            System.out.print("You: ");
            clientMsg = userIn.readLine();
            out.println(clientMsg);
            if (clientMsg.equalsIgnoreCase("quit")) {
                System.out.println("Disconnected from the server.");
                break;
            }

            serverMsg = in.readLine();
            System.out.println("Server: " + serverMsg);
        }

        socket.close();
    }
}

File: prev codes/tcpclient.py
import socket 

client = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
client.connect(("127.0.0.1", 5001))

while True:
    clientMsg = input("You: ")
    client.sendall(clientMsg.encode('utf-8'))

    if clientMsg == 'quit':
        print('disconnected')
        break

    serverMsg = client.recv(1024).decode('utf-8')
    print(f"server: {serverMsg}")
client.close()